src/components/profile/userProfile.jsx

```diff
@@ -5,54 +5,86 @@ import Box from '@material-ui/core/Box';
 import Typography from '@material-ui/core/Typography';
 import { Link } from 'react-router-dom';
 import { socketManagement } from '../../actions/notificationPane';
-
+import ExitToAppIcon from '@material-ui/icons/ExitToApp';
+import IconButton from '@material-ui/core/IconButton';
+import Tooltip from '@material-ui/core/Tooltip';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import { Divider } from '@material-ui/core';
+import { logout, history } from '../../actions/signInAction';
+import Navlinks from './Nav.links.jsx';
 
 export class UserProfile extends Component {
 	// getting a loged-in user information
 	UNSAFE_componentWillMount() {
 		this.props.GetUserProfile();
 	}
+	state = {
+		logout: false,
```

Comment:
@Jajabenit250 any reason to keep the state of logout ? What's the rationale behind this?

Reply:
@bdusimi i did that for tests because always we have to expect some change that's why i was avoiding feedback of no assertion in tests..